mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/quota/CassandraPerUserMaxQuotaManager.java

```diff
@@ -65,32 +66,32 @@ public void setMaxMessage(QuotaRoot quotaRoot, QuotaCount maxMessageCount) {
 
     @Override
     public void setDomainMaxMessage(String domain, QuotaCount count) {
-        perDomainQuota.setMaxMessage(domain, count);
+        perDomainQuota.setMaxMessage(domain.toLowerCase(Locale.US), count);
```

Comment:
I'm against this changeset : given the amount of duplication, it's time to introduce a domain object

Reply:
I will remove it from this PR and open an new PR about that